app/api/register/route.ts

```diff
@@ -1,15 +1,39 @@
 import { NextResponse } from "next/server";
+import { createClient } from '@supabase/supabase-js';
 // for sending an email verification message
 // import crypto from 'crypto';
 // import { sendVerificationEmail } from '@/app/lib/sendEmail';
 
+// Start the supabase client
+const supabase = createClient( process.env.SUPABASE_URL, process.env.SUPABASE_ANON_KEY);
```

Comment:
```suggestion
const supabase = createClient( process.env.SUPABASE_URL || "", process.env.SUPABASE_ANON_KEY || "" );
```
adding ` || ""` ensures an invalid key if the environment variable doesn't exist